### PR TITLE
subsino/subsino.cpp: Dumped and added another New HUNTer set (as MNW)

### PR DESCRIPTION
**Pull request by clawgrip**
New NOT_WORKING clone
------------------------------
New HUNTer (bootleg, set 2) [Taksangs]

### CODE REVIEW

File: src/mame/subsino/subsino.cpp [hunk 1]
```diff
@@ -3358,6 +3358,53 @@ ROM_START( newhunterb )
...
 
+/*  Several custom modifications on this PCB:
+     -3V battery directly soldered to the main HM62256ALP-10 RAM.
+     -One 8-dipswitches bank soldered to a 74LS245.
+     -One 5-dipswitches bank soldered to another 74LS245.
+     -A single switch button soldered to another 74LS245.
+   And the usual hardware configuration:
+     -HD647180X0CP6.
+     -12 MHz xtal.
+     -1 x 8-dipswitches bank, plus the other two banks described before.
```
Comment (cuavas) [inline src/mame/subsino/subsino.cpp:3369]: Can you please write “DIP switches” in plain English?  “One bank of 8 DIP switches…” etc.

Comment (clawgrip) [reply src/mame/subsino/subsino.cpp]: Fixed, thanks!

**Comment by ajrhacker** [comment]
Were the color PROMs actually dumped for this set, or just copied over? The tiles for this set seem to belong to a quite different palette from the other clones.

**Comment by clawgrip** [comment]
No, my bad, I forgot to mark them.

I've opened a PR to flag them as BAD_DUMP: #11641 

Sorry!
